.github/CODEOWNERS

```diff
@@ -8,8 +8,14 @@
 
 # These owners will be the default owners for everything in
 # the repo. Unless a later match takes precedence, members
-# of @climatepolicyradar/tech-devs will be requested for
+# of @climatepolicyradar/software will be requested for
 # review when someone opens a pull request. Teams should
 # be identified in the format @org/team-name. Teams must have
 # explicit write access to the repository.
-* @climatepolicyradar/tech-devs
+
+* @climatepolicyradar/software
+
+app/api/api_v1/routers/pipeline_trigger.py     @climatepolicyradar/deng
```

Comment:
Can we make the search schema file `deng` as well please, Jesse? I feel like a once over from one of you on that would be beneficial. The search router is probably us or you or both, so maybe `tech-devs` there until the application team have more familiarity with the search code 🤔

Reply:
@katybaulch: Done! Do you mind double-checking if I got the right schema file(s) and directory(s)?

Reply:
Almost! can we add `app/api/api_v1/schemas/search.py @climatepolicyradar/deng` so that file is explicitly requires an approval from platform please? :)

Reply:
✅ Thank you for double-checking!